components/match2/commons/match_summary_ffa.lua

```diff
@@ -107,6 +107,7 @@ local MATCH_OVERVIEW_COLUMNS = {
 					showLink = true,
 					overflow = 'ellipsis',
 					teamStyle = 'hybrid',
+					showPlayerTeam = true,
```

Comment:
fwiw not all wikis will want to show team icons there

Reply:
I think we should do it, it's enough space for it